api/openapi/specs/common/error_schema.yaml

```diff
@@ -54,4 +54,26 @@ components:
           type: array
           description: TODO
           items:
-            $ref: "#/components/schemas/InvalidParameters"
\ No newline at end of file
+            $ref: "#/components/schemas/InvalidParameters"
+    NotFoundError:
+      allOf:
+        - $ref: '#/components/schemas/Error'
+        - type: object
+          properties:
+            status:
+              example: 404
+            title:
+              example: Not Found
+            type:
+              example: https://httpstatuses.com/404
+            instance:
+              example: kong:trace:1234567890
+            detail:
+              example: Not found
```

Comment:
Do we need to supplement `type: integer/string` for those properties?

Reply:
This is taken directly from common (platform) definitions, in order for this to properly dedup I can't change anything.